index.html

```diff
@@ -26,7 +26,12 @@ <h1>Welcome to the student showcase!</h1>
         <div>SIX</div>
         <div>SEVEN</div>
         <div>EIGHT</div>
-        <div>NINE</div>
+        <div> 
+            <h3>Fabiane Clubb</h3> 
+           <img src="https://media.licdn.com/dms/image/C4E03AQGeXeHQcJNl0w/profile-displayphoto-shrink_200_200/0?e=1546473600&v=beta&t=l1YTiid7CyOnu_54qqqm-U9ZcL_yWSC8fIEw0sJzAZg"/> 
+            <h1>"The more you weigh, the harder to kidnap. Eat Cake".</h1>
```

Comment:
remember that `h1` is supposed to be the largest text on the page! Let's reserve that for the title of the whole webpage.